app/controllers/redirect/marketing_site_controller.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Redirect
+  class MarketingSiteController < RedirectController
+    def show
+      redirect_to_and_log(MarketingSite.base_url)
+    end
+  end
+end
```

Comment:
Part of the story here was that we want to redirect to straight "www.login.gov" if the user clicks "Exit Login.gov", and log an event. The most pragmatic way of doing this I saw was to just add a new controller, rather than weirdly cramming it into the HelpCenterController.

Reply:
Makes sense!